backend/authschemes/auth_bridge_test.go

```diff
@@ -59,58 +60,40 @@ func TestCreateNewUser(t *testing.T) {
 
 type testSession struct{ Some string }
 
+func createContextForRequest(sessionManager scs.SessionManager, r *http.Request, w http.ResponseWriter) *http.Request {
+	var token string
+	cookie, err := r.Cookie("cookie-name")
+	if err == nil {
+		token = cookie.Value
+	}
+
+	ctx, err := sessionManager.Load(r.Context(), token)
+	if err != nil {
+		sessionManager.ErrorFunc(w, r, err)
+		return r
+	}
+	return r.WithContext(ctx)
+}
+
 func TestLoginUser(t *testing.T) {
-	_, sessionStore, bridge := initBridgeTest(t)
+	_, sessionManager, bridge := initBridgeTest(t)
 
 	gob.Register(&testSession{})
 
 	userID := createDummyUser(t, bridge, "")
 
 	browser := &testBrowser{}
 	w, r := browser.newRequest()
-	err := bridge.LoginUser(w, r, userID, &testSession{Some: "data"})
+	sr := createContextForRequest(*sessionManager, r, w)
+	err := bridge.LoginUser(w, sr, userID, &testSession{Some: "data"})
 	require.NoError(t, err)
 
-	_, r = browser.newRequest()
-	session := sessionStore.Read(r)
+	session := session.GetSession(sessionManager, sr)
 	require.NoError(t, err)
 	require.Equal(t, userID, session.UserID)
 	require.Equal(t, "data", session.AuthSchemeData.(*testSession).Some)
 }
 
-func TestAddToSession(t *testing.T) {
```

Comment:
given that `bridge.SetAuthSchemeSession` is a wrapper for an scs function that doesn't return anything, there was really no reason to keep this test